exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/VarLenColumnBulkInput.java

```diff
@@ -109,6 +109,8 @@
   /** {@inheritDoc} */
   @Override
   public boolean hasNext() {
+    checkCancellation(); // Checks whether query cancellation has been called
```

Comment:
consider `checkInterrupted`

Reply:
do you mean rename the method to checkInterrupted?